test/fixtures/content-sidebar/restored-file.json

```diff
@@ -0,0 +1,102 @@
+{
```

Comment:
Since this is a read operation, can we avoid mocking out the response? Or is there a concern that the file will accidentally get modified?

Reply:
Since we're using the "versioned" file I think it's reasonable for someone to add a new version at some point. Since this logic only works for when current versions are restored it would break the tests.